lib/syskit/roby_app/plugin.rb

```diff
@@ -942,6 +957,32 @@ def self.setup_rest_interface(app, rest_api)
                 require "syskit/roby_app/rest_api"
                 rest_api.mount REST_API => "/syskit"
             end
+
+            def rotate_logs(rotated_logs)
+                plan.find_tasks(Syskit::LoggerService).running.each do |task|
+                    rotated_logs[task.execution_agent.on] = task.rotate_log
```

Comment:
This assumes that there is only one task for any given remote process. Which is rather obviously wrong.

- write a separate test for this (which should fail)
- then fix the method